pkg/controller/controller.go

```diff
@@ -187,6 +191,24 @@ func (c *Controller) ensureK8sResources(funcObj *kubelessApi.Function) error {
 	}
 	funcObj.ObjectMeta.Labels["function"] = funcObj.ObjectMeta.Name
 
+	controllerNamespace := os.Getenv("KUBELESS_INSTALLED_NAMESPACE")
```

Comment:
did you update the deployment yaml of kubeless controller ?

Reply:
@rakesh-garimella yes, pls have a look at `kubeless-rbac.jsonnet`

Reply:
@rakesh-garimella added in `kubeless.jsonnet`